x-pack/plugins/secops/public/components/page/hosts/authorization_table/index.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge } from '@elastic/eui';
+import { defaultTo } from 'lodash/fp';
+import React from 'react';
+import { connect } from 'react-redux';
+import { pure } from 'recompose';
+
+import moment from 'moment';
+import { AuthorizationItem, AuthorizationsEdges } from '../../../../graphql/types';
+import { authorizationsLimitSelector, hostsActions, State } from '../../../../store';
+import { ItemsPerRow, LoadMoreTable } from '../../../load_more_table';
+
+interface OwnProps {
+  data: AuthorizationsEdges[];
+  loading: boolean;
+  hasNextPage: boolean;
+  nextCursor: string;
+  totalCount: number;
+  loadMore: (cursor: string) => void;
+}
+
+interface AuthorizationTableReduxProps {
+  limit: number;
+}
+
+interface AuthorizationTableDispatchProps {
+  updateLimitPagination: (param: { limit: number }) => void;
+}
+
+type AuthorizationTableProps = OwnProps &
+  AuthorizationTableReduxProps &
+  AuthorizationTableDispatchProps;
+
+const rowItems: ItemsPerRow[] = [
+  {
+    text: '2 rows',
```

Comment:
It feels weird to me to offer the option to only show two rows in this table. What's the benefit?

Reply:
That was just for testing when we only add one table with the `loadmore` functionality with a cursor pagination

Reply:
It can be similar to the other ones now!!!

Reply:
I just removed the `2`. Probably we will need to consolidate all of them into one place if we want uniformity among all of them.